lib/packages/core/lib/src/views/home/home_feed.dart

```diff
@@ -95,6 +96,8 @@ class _LMChatHomeScreenState extends State<LMChatHomeScreen> {
     if (state is LMChatHomeLoaded) {
       List<LMChatTile> chatItems = getChats(context, state.response);
       _pageKey++;
+      homeFeedPagingController.itemList?.clear();
```

Comment:
This step might break the pagination flow, Please test it by creating 10 chatrooms and with pagesize 2 once